Topic 3/src/main/java/bootcamp/exercises/ConvertToText.java

```diff
@@ -0,0 +1,128 @@
+package bootcamp.exercises;
+
+public class ConvertToText {
+	 private final String[] UNITS = {"", "one", "two", "three", "four","five","six","seven","eight","nine"};
+
+	  private final String[] TENS = {"ten","eleven","twelve","thirteen","fourteen","fifteen","sixteen","seventeen","eighteen","nineteen", "twenty", "thirty","forty","fifty","sixty","seventy","eighty","ninety"};
+
+		  
+	  public String convert(String number) {
+		  String intPart= "";
+		  String decimalPart;
+		  number= number.replace(".", ",");
+		  if(number.indexOf(",")==-1){
+			  number= number+ ",00";
+		  }
+		  String Num[]= number.split(",");
+		  decimalPart= Num[1];
+		  int lenght= Num[0].length();
+		  int lenght2= decimalPart.length();
+		  if((lenght< 10) && (lenght2 <3)){
+			  decimalPart="and " + decimalPart + "/100 dollars.";
+			  String integerPart= Num[0];
+			  if(Integer.parseInt(integerPart) == 0){
```

Comment:
[High] This will throw a <code>java.lang.NumberFormatException</code> if the user enters an invalid number. For example, 345a5,98b.
You should validate those scenarios also